azure-pipelines.yml

```diff
@@ -1,6 +1,3 @@
-# Copyright lowRISC contributors.
```

Comment:
This file needs a licence, I think. If i excluded it in the config, that was potentially a mistake.

Reply:
Oh duuuur I see what you're doing.

Reply:
That change was the last commit in the series (called "bogus"): the whole point was to check that it failed CI properly. Unfortunately, I couldn't get CI to run without marking as ready for a review :-/ Anyway, I've dropped that now, and all should be well!